docs/product-user-guide/src/main/asciidoc/data-sets-caching-con.adoc

```diff
@@ -13,11 +13,15 @@
 
 When this cache is turned on, the data set is cached in a web browser during the look-up operation. Consequently, further look-up operations do not perform requests to the back end. Data set operations like grouping, aggregations, filtering and sorting are processed in the web browser. Enable client caching only if the data set size is small (for example, less than 10 mb) otherwise for large data sets, browser issues such as slow performance or intermittent freezing can occur.
 
+You should use client caching if you determine that the data set size of your project is small. Client caching reduces the the number of back-end requests including requests to the storage system.
```

Comment:
> _You should use client caching if you determine that the data set size of your project is small._

This is exactly the same as _Enable client caching only if the data set size is small..._ written above.

Reply:
Change acknowledged.